prebuild_binaries/build_scripts/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
Typically bash scripts include a shebang line like "#!/bin/bash" as the first line.

Reply:
I think I have this?

Reply:
I swear I didn't see it. Ignore then.